js/popup_survey.js

```diff
@@ -6,12 +6,18 @@
 
   $(document).ready(function() {
     centrePopup();
+    var bots = /alexa|bot|crawl|facebookexternalhit|feedburner|google web preview|nagios|postrank|pingdom|slurp|spider|yahoo!|yandex|sogou/i;
+
+    if( navigator.userAgent.match(bots) ) {
```

Comment:
Only thing I would add is to check that navigator.userAgent isn't undefined. I remember something about that being deprecated or changing or something...